src/io/aviso/taxi_toolkit/ui.clj

```diff
@@ -51,24 +55,30 @@
 (defn a-click
   "Element-agnostic. Runs either (taxi/click) or (click-anything)."
   [& el-spec]
-  (let [el (apply $ el-spec)]
-    (case (.getTagName (:webelement el))
-      ("a" "button") (retry #(click el))
-      (retry #(click-non-clickable el)))))
+  (retrying
+   (let [el (apply $ el-spec)]
+     (case (.getTagName (:webelement el))
+       ("a" "button") (click el)
+       (click-non-clickable el)))))
 
 (defn a-text
   "For non-form elements such as <div> works like (taxi/text).
   For <input> works like (taxi/value)."
   [el]
-  (case (.getTagName (:webelement el))
-    ("input") (retry #(value el))
-    (retry #(text el))))
+  ;; TODO: this one is sometimes used as a checker in smart-wait/wait-until
+  ;; and sometimes as a standalone operation. The retrying here is not needed
+  ;; in case of smart-wait
+  (retrying
```

Comment:
I'm thinking that whatever calls this should either use the spec, either retry both the find and this ina *that* place. Maybe there are few instances, in any case it's worth giving it a shot. O/w don't bother reverting.